keps/sig-instrumentation/3498-extending-stability/README.md

```diff
@@ -203,6 +209,34 @@ We will not attempt to parse metrics which:
 
 As an aside, much of this work has already been done, but is stashed in a local repo. 
 
+### Semantic of Stability Levels
+
+#### Internal Metrics
+
+`Internal` metrics have no stability guarantees and are **not** parseable by the static analysis framework. As such, `Internal` metrics will NOT be included in metric auto-documentation.
+
+#### Alpha Metrics
+
+Alpha metrics have no stability guarantees but are parseable by the static analysis framework. As such, `Alpha` metrics will be included in metric auto-documentation.
+
+#### Beta Metrics
+
+`Beta` metrics have *some* stability guarantees. Specifically, we guarantee that:
+
+ - `Beta` metrics will not be removed without first being explicitly deprecated. 
+    + you can deprecate Beta metrics at any point if because of changes in underlying code/feature (i.e. it's impossible to compute such metric with the code changes, you can remove it after one release)
```

Comment:
Thanks for offline discussion about that and getting to a compromise here.

I'm fine with the message being passed here, but let's rephrase - it's slightly hard to parse for me now.
My suggestion is:
```
+ you can deprecate Beta metrics at any point
+ if because of changes in underlying code/feature it's impossible to compute such metric the metric can be removed after one release
+ if the metric is still ...
```